js/navigation.js

```diff
@@ -18,6 +22,18 @@ class NavigationController extends Backbone.Controller {
     Adapt.navigation = new NavigationView();// This should be triggered after 'app:dataReady' as plugins might want to manipulate the navigation
   }
 
+  onDeviceResize() {
+    const adaptConfig = Adapt.course.get('_navigation');
+    const $html = $('html');
+    $html.addClass('is-nav-top');
+    if (!adaptConfig?._navigationAlignment) return;
+    let navigationAlignment = adaptConfig._navigationAlignment;
+    if (adaptConfig._isBottomOnTouchDevices && device.touch) {
```

Comment:
This doesn't seem right to me. E.g. If you wanted bottom navigation on mobile only, this isn't configurable in the JSON due to the guard clause on line 29.